### PR TITLE
find: add maxdepth in example

### LINKED ISSUE #8540: find: mention maxdepth 1
Very useful option

### PR DESCRIPTION
**Pull request by fallenpixel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Closes #8540

### CODE REVIEW

File: pages/common/find.md [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 `find {{root_path}} -name '{{*.py}}' -not -path '{{*/site-packages/*}}'`
 
-- Find files matching a given size range:
+- Find files matching a given size range excluding subdirectories:
```
Comment (kbdharun) [inline pages/common/find.md:22]:
```suggestion
- Find files matching a given size range, excluding subdirectories:
```
To be consistent with the above description.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a suggestion for this page.

> Also, Welcome to tldr btw.

File: pages/common/find.md (hunk 1 above)
Comment (fallenpixel) [reply pages/common/find.md]: [fc92c71](https://github.com/tldr-pages/tldr/pull/8719/commits/fc92c71753157c7250524c7f56ccc4e5587301db) includes this suggestion. Thank you.

**Comment by CleanMachine1** [comment]
Not sure about this change. Personally, it seems like a strange addition from a practical view of the command, if I want to search for large files, why would I want to only search the root directory of the search and not the sub-directories.

Maybe this would fit into a different example.

File: pages/common/find.md [hunk 2]
```diff
@@ -19,9 +19,9 @@
 
 `find {{root_path}} -name '{{*.py}}' -not -path '{{*/site-packages/*}}'`
 
-- Find files matching a given size range:
+- Find files matching a given size range, excluding subdirectories:
```
[the remark this answers is not in the document]
Comment (fallenpixel) [reply pages/common/find.md]: Better wording, Thank you.

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you @fallenpixel for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**